core/embed/models/model_T1B1_layout.c

```diff
@@ -0,0 +1,39 @@
+#include "flash.h"
+#include "model.h"
+
+const flash_area_t STORAGE_AREAS[STORAGE_AREAS_COUNT] = {
+    {
+        .num_subareas = 1,
+        .subarea[0] =
+            {
+                .first_sector = 2,
+                .num_sectors = 1,
+            },
+    },
+    {
+        .num_subareas = 1,
+        .subarea[0] =
+            {
+                .first_sector = 3,
+                .num_sectors = 1,
+            },
+    },
+};
+
+const flash_area_t BOOTLOADER_AREA = {
+    .num_subareas = 1,
+    .subarea[0] =
+        {
+            .first_sector = 5,
+            .num_sectors = 1,
+        },
+};
+
+const flash_area_t FIRMWARE_AREA = {
+    .num_subareas = 1,
+    .subarea[0] =
+        {
+            .first_sector = 6,
+            .num_sectors = 6,
```

Comment:
`.num_sectors = 7`?

Reply:
hmm i adjusted what was in flash.h, seemed that it wasn't set correctly there for T1 at all: https://github.com/trezor/trezor-firmware/commit/df115b63de11c2f18e4c4e9219ebfd9feebaf028